test/e2e_node/cpu_manager_test.go

```diff
@@ -710,6 +774,21 @@ func runCPUManagerTests(f *framework.Framework) {
 		runSMTAlignmentPositiveTests(ctx, f, smtLevel)
 	})
 
+	ginkgo.It("should not enforce CFS quota for containers with static CPUs assigned", func(ctx context.Context) {
+
+		newCfg := configureCPUManagerInKubelet(oldCfg,
+			&cpuManagerKubeletArguments{
+				policyName:                       string(cpumanager.PolicyStatic),
+				reservedSystemCPUs:               cpuset.New(0),
+				enableCPUManagerOptions:          true,
+				disableCPUQuotaWithExclusiveCPUs: true,
```

Comment:
we would like another test (please pick the most fitting place here it fits, or feel free to create a new one if noone really fits) where we check that with the feature gate is disabled, no pod gets unlimited quota. Basically let's run all the cases you added but with the feature gate forced to off to ensure and verify users can get the old behavior.

Once that is done, the PR is ready.

Reply:
done!